task_6_2.py

```diff
@@ -0,0 +1,19 @@
+FILENAME = 'result.txt'
+
+
+def check_even_odd(num):
```

Comment:
def check_even_odd(num):
    if not num:
        return f'Number {num} - {"odd" if num % 2 == 1 else "even"}.'
    return 'neither even nor odd'